src/engine/strat_engine/metadata/bda.rs

```diff
@@ -60,7 +60,8 @@ impl BDA {
     where
         F: Read + Seek + SyncAll,
     {
-        let header = match StaticHeader::setup(f)? {
+        let read_results = StaticHeader::read_sigblocks(f);
```

Comment:
The code all looks good. The best thing to do is to consider this method, ```BDA::load```. If it's being called by stratis_dumpmetdata what should it do, instead of what it is doing right now? It might be worthwhile to write that out in words, and then we can think about how we should implement our decision in code.